spec/helpers/product_test_data.rb

```diff
@@ -1,18 +1,18 @@
 module Katello
   module ProductTestData
-    ORG_ID = "admin-org-37070".freeze
-    PRODUCT_ID = '37070'.freeze
-    PRODUCTS =
+    ORG_ID ||= "admin-org-37070".freeze
+    PRODUCT_ID ||= '37070'.freeze
+    PRODUCTS ||=
```

Comment:
curious, what does this solve? (reinitialized constant warnings?)

Reply:
Yes, this is for the reinitialized constants warnings.